rspace/src/test/scala/coop/rchain/rspace/test/InMemoryStore.scala

```diff
@@ -24,9 +21,16 @@ class InMemoryStore[C, P, A, K] private (
 
   val eventsCounter: StoreEventsCounter = new StoreEventsCounter()
 
+  private[this] var _dbGNATs: Map[Blake2b256Hash, GNAT[C, P, A, K]] = Map.empty
+  private[this] var _dbJoins: Map[C, Seq[Seq[C]]]                   = Map.empty
+
+  private[rspace] type H = Blake2b256Hash
```

Comment:
Now that this is gone from `IStore`, let's not bring this back here.  I'd prefer to have `Blake2b256Hash` inlined method signatures.